### ISSUE TITLE
Button work incorrect when have loading and disabled props

### ISSUE THREAD

**Issue by tuduong** [jsdir/react-ladda#25]
`<LaddaButton buttonStyle="zoom-in" className={ classes }
    loading={ this.state.saving }
    onClick={ this.save }
    disabled={ !hasChanged || this.state.saving }>
    Save
</LaddaButton>`

Recommend:
add re check disabled prop when stoped ladda button
```
else if (!this.props.loading && this.laddaButton.isLoading()){
    this.laddaButton.stop();
    if (this.props.disabled) {
	    this.laddaButton.disable();
    }
}
```

**Comment by jsdir** [comment]
@tuduong This has been fixed in `react-ladda@4.0.2`

**Comment by tuduong** [comment]
Thanks
